app/views/brexit_checker/results.html.erb

```diff
@@ -1,19 +1,7 @@
 <%
-  action_based_email_link_label = @actions.any? ? t('brexit_checker.results.email_sign_up_link') : t('brexit_checker.results.email_sign_up_link_no_actions')
-  action_based_title            = if @actions.any?
-                                    t('brexit_checker.results.title')
-                                  elsif criteria_keys.present?
-                                    t('brexit_checker.results.title_no_actions')
-                                  else
-                                    t('brexit_checker.results.title_no_answers')
-                                  end
-  action_based_description      = if @actions.any?
-                                    t('brexit_checker.results.description')
-                                  elsif criteria_keys.present?
-                                    t('brexit_checker.results.description_no_actions')
-                                  else
-                                    t('brexit_checker.results.description_no_answers')
-                                  end
+  action_based_email_link_label = email_link_label(@actions)
+  action_based_title            = title(@actions, criteria_keys)
```

Comment:
Minor, but I find this kind of spacing irritating when I need to make a change, because it tends to cause a lot of extra typing. We don't tend to do this anywhere else (probably for that reason).

Reply:
Yeah wasn't sure about that, largely just went with the flow as it was there.
If that's not something we do elsewhere then i don't want it either.

Removed and pushed amended change.

Reply:
Ah yeah. Sorry, I didn't check the "before" 😬